### DISCUSSION TITLE
Routing bug on documentation page

### DISCUSSION THREAD

**Discussion by zhebil** [gatsbyjs/gatsby#31584] [Help]
## Description

Site content disappears, and the following error appears in the console:
![image](https://user-images.githubusercontent.com/63890293/119504203-6dbeaa00-bd74-11eb-8e51-f5fe3149132c.png)

### Steps to reproduce
1. Open the documentation - https://www.gatsbyjs.com/docs/
2. I open the context menu by clicking on the right mouse button
3. I translate the page into Russian by clicking on the item "Translate into Russian"
4. I click on the item in the drop-down list on the right of the page - "Tutorial" (or any other)

### Expected result

The "Tutorial" page opens.

### Actual result

I see a blank page. There is an error in the console (screen attached)

### Environment

MacBook Pro (Retina, 13-inch, Mid 2014)
macOS Big Sur 11.3.1
Google Chrom 90.0.4430.212

**Comment by LekoArts** [comment]
I followed your steps and don't see this behavior. The routing works fine for me
